de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/editing/EditDifferentFilesTest.java

```diff
@@ -94,10 +94,12 @@ public void run() throws Exception {
             .getFileContent("foo/src/bar/HelloWorld.java");
 
         String contentAliceHelloGermany = ALICE.superBot().views()
-            .packageExplorerView().getFileContent("foo/src/HelloGermany.java");
+            .packageExplorerView()
+            .getFileContent("foo/src/bar/HelloGermany.java");
```

Comment:
This test failed, because the test tried to get the file content of a file that does not exist ("foo/src/HelloGermany.java")

Reply:
And why does the file do not exist ?

Reply:
The default behaviour of the class creation wizard changed. Instead of using the default package an existing package is used. I see my solution is no very good. Instead of changing the `getFileContent` I will change the `cls(String class)` method to use always the default package. Using the default eclipse behaviour in `cls` is error prone.